drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioSimulationServiceImpl.java

```diff
@@ -122,6 +122,7 @@
     private Properties props = new Properties();
 
     private static final String KIE_VERSION = "kie.version";
+    private static final String junitActivatorPackageName = "test";
```

Comment:
@danielezonca 
Do not use "test" as package name, since it is confusing and it is a word too easily used elsewhere.
Whenever there is the need to create an internal-use only keyword, is always better to use something really specific.

Reply:
What about `testscenario`?

Reply:
Done